examples/get_event_messages.c

```diff
@@ -10,6 +10,11 @@
 
 bool is_error = false;
 
+#define milestones_latest_topic "milestones/latest"
+#define milestones_confirmed_topic "milestones/confirmed"
+#define messages_referenced_topic "messages/referenced"
```

Comment:
imo, it's often used in lib and application. topics should define in `client/api/events/node_event.h`
something like
```
#define TOPIC_MS_LATEST "milestones/latest"
#define TOPIC_MS_CONFIRMED "milestones/confirmed"
...
```

Reply:
Done.